src/Tribe/Editor/Configuration.php

```diff
@@ -15,6 +16,45 @@ public function hook() {
 		add_filter( 'tribe_editor_config', array( $this, 'editor_config' ) );
 	}
 
+	/**
+	 * Setup the initial Event Object.
+	 *
+	 * @since TBD
+	 *
+	 * @todo @backend Improve this method and verify if this is the correct place for this to live.
+	 *
+	 * @param null|int|WP_Post $event
+	 *
+	 * @return array Setup the Data for the Event Post Object.
+	 */
+	public function get_event_object( $event = null ) {
+		$data = [
+			'is_new_post' => true,
+		];
+
+		if ( ! $event ) {
+			$event = tribe_get_request_var( 'post', false );
+		}
+
+		if ( $event ) {
+			$event = get_post( $event );
+		}
+
+		if ( ! $event instanceof  WP_Post) {
+			return $data;
+		}
+
+		$meta = get_post_meta( $event->ID );
+		$meta = array_map( static function( $value ) {
+			return count( $value ) === 1 ? reset( $value ) : $value;
+		}, $meta );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 3e1efe2ff34e5f4d228ca36bb1290cde</sub>